eng/pipelines/common/global-build-job.yml

```diff
@@ -75,18 +75,19 @@ jobs:
       - name: _osParameter
         value: -os ${{ parameters.osGroup }}
 
-      - ${{ if and(eq(parameters.osGroup, 'Linux'), eq(parameters.osSubGroup, '_musl')) }}:
-        - name: _osParameter
-          value: /p:RuntimeOS=linux-musl /p:OutputRid=linux-musl-${{ parameters.archType }}
-
       - ${{ if and(eq(parameters.osGroup, 'Linux'), eq(parameters.osSubGroup, '_bionic')) }}:
         - name: _osParameter
-          value: /p:RuntimeOS=linux-bionic /p:OutputRid=linux-bionic-${{ parameters.archType }}
+          value: /p:RuntimeOS=linux-bionic
```

Comment:
Is the arch no longer needed here?

Reply:
Yup. It is redundant: linux with bionic-libc is the only OS we cannot automatically recognize; for musl, we used [this hack](https://github.com/dotnet/runtime/blob/main/eng/native/init-distro-rid.sh#L163) but there doesn't seem to be anything reliable in case of bionic. That's why we need this external hint in the form of `RuntimeOS`. Otherwise, all the derived properties including `OutputRid` don't need external hints as for any other platform in the matrix.